packages/frontend/web/components/Footer.tsx

```diff
@@ -142,47 +147,28 @@ const footerList = css`
 
         ${desktop} {
             :nth-of-type(1) {
-                border-left: 1px solid ${palette.brand.pastel};
+                border-left: ${footerBorders};
             }
         }
     }
 `;
 
 const readerRevenueLinks = css`
-    border-left: 1px solid ${palette.brand.pastel};
+    border-left: ${footerBorders};
     flex: 1;
     padding: 12px 0 0 10px;
     margin: 0 10px 36px 0;
     width: calc(50% - 10px);
 
     ${until.tablet} {
         width: 50%;
-        border-top: 1px solid ${palette.brand.pastel};
+        border-top: ${footerBorders};
     }
 `;
 
 const copyright = css`
     ${textSans(1)};
-    padding: 0 19px 12px;
-    margin: 12px auto 0;
-
-    ${tablet} {
-        max-width: 740px;
-    }
-
-    ${desktop} {
-        max-width: 980px;
-    }
-
-    ${leftCol} {
-        max-width: 1140px;
-    }
-
-    ${wide} {
-        max-width: 1300px;
-    }
-
-    display: block;
+    padding: 12px;
```

Comment:
Whole lot got removed because I shifted the copyright into the container. Used all paddings instead of margins and relied on widths and default displays doing their thing.